index.html

```diff
@@ -218,6 +237,400 @@ <h2>
+          </p>
+        </section>
+      </section>
+      <section>
+        <h3>
+          <code><dfn>new_tab_button</dfn></code> member
+        </h3>
+        <p>
+          The [=tab_strip/new_tab_button=] member is an ordered map that
+          describes the behaviour of a UI affordance (such as a button) which,
+          when clicked/activated, opens a new [=application context=] within
+          the application window. It has the following members:
+        </p>
+        <ul>
+          <li>[=new_tab_button/url=]
+          </li>
+        </ul>
+        <p>
+          The <code><dfn data-dfn-for="new_tab_button">url</dfn></code> member
+          is a string that represents a URL relative to the [=manifest URL=]
+          that is [=manifest/within scope=] of a [=Document/processed
+          manifest=].
+        </p>
+        <aside class="note">
+          The [=new_tab_button=] member will be ignored if it is [=within home
+          tab scope=].
+        </aside>
+        <p>
+          When the new tab button is activated by the end user, the following
+          steps are run:
+        </p>
+        <ol>
+          <li>Create a new [=application context=] in the current window and
+          focus it.
+          </li>
+          <li>Navigate it to the value of the [=new_tab_button/url=] member of
+          [=new_tab_button=].
+          </li>
+        </ol>
+      </section>
+      <section>
+        <h2>
+          Processing the `tab_strip` member
+        </h2>
+        <p>
+          To <dfn>process the `tab_strip` member</dfn>, given [=ordered map=]
+          |json:ordered map|, [=ordered map=] |manifest:ordered map|, and
+          [=URL=] |manifest URL:URL|, run the following during the
+          <a data-cite="appmanifest#dfn-processing-extension-point-of-web-manifest">
+          extension point</a> in [=processing a manifest=]:
+        </p>
+        <ol>
+          <li>Set |manifest|["tab_strip"] to a new [=ordered map=].
+          </li>
+          <li>If |json|["tab_strip"] does not exist or |json|["tab_strip"] is
+          not an [=ordered map=]:
+            <ol>
+              <li>Set |manifest|["tab_strip"]["new_tab_button"]["url"] to
+              |manifest|["start_url"].
+              </li>
+              <li>Return.
+              </li>
+            </ol>
+          </li>
+          <li>[=Process the `home_tab` member=] passing |json|["tab_strip"],
+          |manifest|["tab_strip"], and |manifest URL|.
+          </li>
+          <li>[=Process the `new_tab_button` member=] passing
+          |json|["tab_strip"], |manifest|["tab_strip"], |manifest URL|, and
+          |manifest|["start_url"].
+          </li>
+        </ol>
+        <section>
+          <h3>
+            Processing the `home_tab` member
+          </h3>
+          <p>
+            To <dfn>process the `home_tab` member</dfn>, given [=ordered map=]
+            |json:ordered map|, [=ordered map=] |manifest tab strip:ordered
```

Comment:
maybe rename this here and elsewhere `manifest_tab_strip_member`?

Or- not sure what is 'normal' for spec - just wanted to make it clearer this was a variable that is in the parsed manifest output. Fine to keep as-is if it matches convension.

Reply:
Unclear if this is following any other established pattern. (Most of the "complex" processing steps I can see do not take a manifest member, they just return one as a result. We could do that here, but this way is more flexible and follows the top-level better, so I will choose to keep it unless you object.

It is quite normal to have variable names (in spec) with spaces in them, not underscores.